pom.xml

```diff
@@ -29,6 +29,14 @@
       <relativePath>org.apache:apache</relativePath>
    </parent>
 
+   <repositories>
+      <repository>
+         <name>Apache Staging Repository</name>
+         <id>apache.staging</id>
+         <url>https://repository.apache.org/content/groups/staging/</url>
+      </repository>
+   </repositories>
```

Comment:
Would configure this for releases-only so that it isnt ever checked for snapshots (should anyone use this overall change against development commits at a later point, it likely will be)

Maybe add a TODO:delete comment to make super clear it is temp and easy to find later.